apps/web/env.sh

```diff
@@ -5,34 +5,34 @@ rm -rf ./env-config.js
 touch ./env-config.js
 
 # Add assignment
-echo "window._env_ = {" >> ./env-config.js
+echo "window._env_ = {" >>./env-config.js
```

Comment:
We should do exactly the same also for the `widget` app. Could you adjust that file too?